scripts/bug1764246.pl

```diff
@@ -0,0 +1,117 @@
+#!/usr/bin/env perl
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+#
+# This Source Code Form is "Incompatible With Secondary Licenses", as
+# defined by the Mozilla Public License, v. 2.0.
+
+use 5.10.1;
+use strict;
+use warnings;
+use lib qw(. lib local/lib/perl5);
+
+use Bugzilla;
+use Bugzilla::Bug;
+use Bugzilla::Constants;
+use Bugzilla::Group;
+use Bugzilla::Search;
+use Getopt::Long;
+use URI qw();
+
+BEGIN { Bugzilla->extensions(); }
+
+# Bug 1764246
+# Use "triaged" keyword to track bug's triaged state instead of severity
+#
+# Adds triaged keyword to open Firefox bugs with a valid severity set.
```

Comment:
It's been a very long time since I wrote a migration script, please pay particular attention to it.